lib/enerscore/response_parser.rb

```diff
@@ -4,24 +4,23 @@ class ResponseParser
 
     def initialize(response)
       @response = response
-      if @response == :network_timeout
+
+      case @response
+      when :network_timeout
```

Comment:
The `response` is only a symbol when an exception is thrown by HTTParty, otherwise it is JSON, so this won't work